roles/download/defaults/main.yml

```diff
@@ -71,7 +71,7 @@ nerdctl_extra_flags: '{%- if containerd_insecure_registries is defined and conta
 kubeadm_version: "{{ kube_version }}"
 crun_version: 1.4.4
 runc_version: v1.1.1
-kata_containers_version: 2.2.3
+kata_containers_version: 2.4.1
```

Comment:
This changes the default version of kata container to 2.4.1, but it adds checksums only for amd64.
That means if Kubespray users try to use kata container on the other CPU architecture, that would be failed by default.
Please add checksums for the other architectures also to avoid such situation.